### PR TITLE
[Translation zh-tw] updates > 2017 > 09 > picture-in-picture.md

### PR DESCRIPTION
**Pull request by limhenry**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/368)

### CODE REVIEW

**Comment by sotayamashita** [comment]
It fails because of `Error: OAuth2 authentication requires a token or key & secret to be set`

```
[10:06:37] Starting 'test:travis-init'...

[10:06:37] 'test:travis-init' errored after 28 ms

[10:06:37] Error: OAuth2 authentication requires a token or key & secret to be set

    at __dirname.authenticate (/home/travis/build/google/WebFundamentals/node_modules/github/lib/index.js:359:23)

    at Gulp.<anonymous> (/home/travis/build/google/WebFundamentals/gulp-tasks/test.js:1305:10)

    at module.exports (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)

    at Gulp.Orchestrator._runTask (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/index.js:273:3)

    at Gulp.Orchestrator._runStep (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/index.js:214:10)

    at Gulp.Orchestrator.start (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/index.js:134:8)

    at /home/travis/build/google/WebFundamentals/node_modules/gulp/bin/gulp.js:129:20

    at nextTickCallbackWith0Args (node.js:489:9)

    at process._tickCallback (node.js:418:13)

    at Function.Module.runMain (module.js:443:11)
```

File: src/content/zh-tw/updates/2017/09/picture-in-picture.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: 讓我們來討論如何使用一個Web API以在桌面創建一個浮動視頻窗口。 #}
+{# wf_blink_components: Blink>Media #}
+
+# 畫中畫 (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Translated by
```
Comment (petele) [inline src/content/zh-tw/updates/2017/09/picture-in-picture.md]: Per our [style guide](https://developers.google.com/web/resources/translations#credit), please put the translated by at the bottom of the article.

File: src/content/zh-tw/updates/2017/09/picture-in-picture.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+{% include "web/_shared/contributors/henrylim.html" %}
+
+自2017年4月起，[Android O的Chrome支持畫中畫](https://developer.android.com/about/versions/oreo/android-8.0.html#opip)。這功能可讓用戶在同個時間，在悬浮窗中的`<video>`元素播放視頻，一邊做其他的事情。
```
Comment (petele) [inline src/content/zh-tw/updates/2017/09/picture-in-picture.md]: Please break lines at [80 chars](https://developers.google.com/style/html-formatting#line-length)

File: src/content/zh-tw/updates/2017/09/picture-in-picture.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      document.exitPictureInPicture()
+      .catch(error => {
+        // 視頻無法退出畫中畫模式
+      });
+    }
+  });
+</script>
+```
+
+注意: 目前沒有任何的游覽器可以運行以上的代碼。
```
Comment (petele) [inline src/content/zh-tw/updates/2017/09/picture-in-picture.md]: [Callouts](https://developers.google.com/web/resources/markdown-syntax#callouts) like `Note:` should NOT be translated, they're handled differently by the markdown processor.

Comment (sotayamashita) [reply src/content/zh-tw/updates/2017/09/picture-in-picture.md]: @limhenry Could you modify it?

@petele Is there translation style guide?

File: src/content/zh-tw/updates/2017/09/picture-in-picture.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: 讓我們來討論如何使用一個Web API以在桌面創建一個浮動視頻窗口。 #}
+{# wf_blink_components: Blink>Media #}
+
+# 畫中畫 (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+自2017年4月起，[Android O的Chrome支持畫中畫](https://developer.android.com/about/versions/oreo/android-8.0.html#opip)。這功能可讓用戶在同個時間，在悬浮窗中的`<video>`元素播放視頻，一邊做其他的事情。
```
Comment (petele) [inline src/content/zh-tw/updates/2017/09/picture-in-picture.md]: Please break lines at [80 chars](https://developers.google.com/style/html-formatting#line-length)

**Pull request merged**